.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
Hey @fabricedesre, could you please do a first review of the entire PR?

I've ended up with editing ```/etc/dbus-1/system.d/avahi-dbus.conf``` to provide any user/user-group based avahi-daemon access approach we may need (explained below in README.md). Once we have some RPi deployment script/whatever instead we may install ```avahi-daemon``` from sources with ```./configure --with-avahi-priv-access-group=any_group```. Or if someday we end up with Yocto there is already ready reciepe for it: http://cgit.openembedded.org/openembedded-core/tree/meta/recipes-connectivity/avahi that configures access as well.